plugin

```diff
@@ -153,6 +153,7 @@
   "iantrich/text-divider-row",
   "ibz0q/better-moment-card",
   "idaho/hassio-trash-card",
+  "igor-panteleev/lovelace-qr-code-card",
```

Comment:
I do not know how you tricked my action, but the readme does not contain any examples on how this looks.
There needs to be images present there.